src/components/TableResize.js

```diff
@@ -94,21 +94,32 @@ class TableResize extends React.Component {
   };
 
   onResizeStart = (id, e) => {
-    this.setState({ isResize: true, id, startPosition: e.clientX });
+    this.setState({ isResize: true, id });
   };
 
   onResizeMove = (id, e) => {
-    const { startPosition, isResize, resizeCoords } = this.state;
-
+    const { isResize, resizeCoords } = this.state;
+    const fixedMinWidth = 100;
+    const idNumber = parseInt(id);
```

Comment:
id is string type.

Reply:
True. It didn't matter before because there weren't any calculations that depended on it being int, but I guess there are now. Let's use the radix argument here, since we can't be too careful with `parseInt`. When radix is not specified, it's behavior can vary.